### PR TITLE
docs: add link to PollSender

### PR DESCRIPTION
**Pull request by zaharidichev**
## Motivation
There was no link to `PollSender` in mpsc::Sender::send docs

## Solution

Add link

Fix: #3611

### CODE REVIEW

File: tokio/src/sync/mpsc/bounded.rs [hunk 1]
```diff
@@ -314,7 +314,8 @@ impl<T> Sender<T> {
         Sender { chan }
     }
 
-    /// Sends a value, waiting until there is capacity.
+    /// Sends a value, waiting until there is capacity. In case you need a
+    /// polling interface, use [`PollSender`].
```
Comment (Darksonn) [inline tokio/src/sync/mpsc/bounded.rs:318]: I would like the link to be in the same place in the docs as where I put the similar thing for `Semaphore`.

File: tokio/src/sync/mpsc/bounded.rs [hunk 2]
```diff
@@ -332,6 +333,7 @@ impl<T> Sender<T> {
     ///
     /// [`close`]: Receiver::close
     /// [`Receiver`]: Receiver
+    /// [`PollSender`]: https://docs.rs/tokio-util/0.6.4/tokio_util/sync/struct.PollSender.html
```
Comment (Darksonn) [inline tokio/src/sync/mpsc/bounded.rs:336]:
```suggestion
    /// [`PollSender`]: https://docs.rs/tokio-util/0.6/tokio_util/sync/struct.PollSender.html
```

Comment (zaharidichev) [reply tokio/src/sync/mpsc/bounded.rs:336]: Why don't we just put /latest/ here instead of concrete version?

Comment (Darksonn) [reply tokio/src/sync/mpsc/bounded.rs:336]: The link may break if we move it or remove it in future versions of `tokio-util`.

Comment (zaharidichev) [reply tokio/src/sync/mpsc/bounded.rs]: Good to know, thanks!

File: tokio/src/sync/mpsc/bounded.rs [hunk 3]
```diff
@@ -309,6 +309,10 @@ impl<T> fmt::Debug for Receiver<T> {
 
 impl<T> Unpin for Receiver<T> {}
 
+/// To use the `Sender` in a poll function, you can use the [`PollSender`]
+/// utility.
+///
+/// [`PollSender`]: https://docs.rs/tokio-util/0.6/tokio_util/sync/struct.PollSender.html
```
Comment (Darksonn) [inline tokio/src/sync/mpsc/bounded.rs:315]: Put it on the struct, not the impl block.

**Review by davidpdrsn — APPROVED** [review]

**Pull request merged**